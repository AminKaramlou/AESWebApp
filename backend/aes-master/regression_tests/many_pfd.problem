2;
A: 1
;
;
1: A
2: A

