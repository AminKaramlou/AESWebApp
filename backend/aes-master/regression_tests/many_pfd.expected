Schedule is feasible
 -  All jobs are allocated to exactly one machine
Schedule does not satisfies user fixed decisions because:
 -  Job A cannot be allocated to multiple machines {1, 2}
Schedule is efficient
 -  All jobs satisfy single and pairwise exchange properties
